### ISSUE TITLE
Event broadcasting is broken as of 0.3.17

### ISSUE THREAD

**Issue by dmitrystas** [metarhia/Impress#598]
For example: adding listener in `\ias\applications\example\init\event.js` so it will work in all instances after `emit` in any. But events are not broadcasting and `on` fires just in save server instance.

``` js
application.backend.on('test', function(data) {
  console.log('Event "test" received by: ' + application.nodeId);
  console.log('  data: ' + api.json.stringify(data));
});

if (application.nodeId == 'S1N1') {
    api.timers.setInterval(function() {
      application.backend.emit('test', { sender: application.nodeId, data: 'data' });
    }, 5000);
}
```

Console output:

```
  JSTP listen on 127.0.0.1:250 by Master(30166/S1N0) Cloud Controller
  HTTP listen on 127.0.0.1:38881 by Worker(30179/S1N1)
  HTTP listen on 127.0.0.1:38882 by Worker(30180/S1N2)
Event "test" received by: S1N1
  data: {"sender":"S1N1","data":"data"}
Event "test" received by: S1N1
  data: {"sender":"S1N1","data":"data"}
...
Event "test" received by: S1N1
  data: {"sender":"S1N1","data":"data"}
```

While expected:

```
> Event "test" received by: S1N1
>   data: {"sender":"S1N1","data":"data"}
> Event "test" received by: **S1N2**
>   data: {"sender":"S1N1","data":"data"}
```

**Comment by tshemsedinov** [comment]
@dmitrystas now IPC (interprocess communication) is broken because moving from Impress RPC (with JSON serialization) to JSTP. I'll close this issue when it will work again.

**Comment by aqrln** [comment]
Fixed via bb427730dbd730b1adaf35c7c81c65ab8cb6845e.
